### PR TITLE
Added honors distinction to profile spec

### PR DESCRIPTION
**Pull request by kevinrathgeber**
Small section but wanted to make sure I'm on the right track before I keep going

### CODE REVIEW

File: application-profile.yml [hunk 1]
```diff
@@ -1026,5 +1026,52 @@ components:
...
+    Honors:
+      type: object
+      properties:
+        honorName: 
+          type: string
+        honorLevel:
+          $ref: "#/components/schemas/HonorLevel"
+        honorGrade:
+          $ref: "#/components/schemas/HonorGrade"
+        # can't they infer from length?
```
Comment (daynekilheffer) [inline application-profile.yml:1038]: agreed, and either way, probably best to not have "count" on each item

File: application-profile.yml [hunk 2]
```diff
@@ -1026,5 +1026,52 @@ components:
           type: string
         highSchoolCredit:
           type: boolean
+    Honors:
+      type: object
+      properties:
+        honorName:
```
Comment (daynekilheffer) [inline application-profile.yml:1032]: I'd drop the prefix from the field name.  It's part of the Honors object so it feels like stuttering

Comment (kevinrathgeber) [reply application-profile.yml:1032]: good call

File: application-profile.yml [hunk 3]
```diff
@@ -572,11 +572,13 @@ components:
             honorsDistinctions:
               type: object
               properties:
+                notReported:
+                  type: boolean
```
Comment (kevinrathgeber) [inline application-profile.yml:576]: moved this out of the `Honors` object into the top level since if there are none reported, that array will be empty

Comment (daynekilheffer) [reply application-profile.yml:576]: is this different than "honors.length == 0"

**Review by daynekilheffer — APPROVED** [review]

**Pull request merged**